test/e2e/test_job_submissions.py

```diff
@@ -1727,3 +1685,54 @@ def session_action_has_expected_progress(session_action_id) -> bool:
         job.wait_until_complete(client=deadline_client)
 
         assert job.task_run_status == TaskStatus.SUCCEEDED
+
+    def test_worker_enters_stopping_state_while_draining(
+        self,
+        deadline_resources: DeadlineResources,
+        deadline_client: DeadlineClient,
+        function_worker: EC2InstanceWorker,
+        sleep_script: str = (
+            """
+            #!/usr/bin/env bash
+            sleep 600
+            """
+            if os.environ["OPERATING_SYSTEM"] == "linux"
+            else """
+            Start-Sleep -Seconds 600
+            """
+        ),
```

Comment:
Does this need to be a function parameter? Can we just define it as a variable in the test?